src/styles.scss

```
@@ -261,3 +267,113 @@ mat-card,
+  background: rgba(255,255,255,0.25);
+}
+
+/* =========================================================
+   ROWS (pseudo table rows)
+========================================================= */
+.rowForm {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(220px, 1fr));
+  gap: 16px;
+
+  padding: 18px;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)